docs/installation.rst

```diff
@@ -6,7 +6,7 @@ Installation guide
 Requirements
 ------------
 
-polib requires python 2.5 or higher.
+polib requires python 3.6 or higher.
```

Comment:
Hi Arnaud, in fact polib supports python2, so this should be 2.7

Reply:
oh right, I saw your last commit and misread the < for a <= since Python 2 has been deprecated since last year it made sense to me :)

Updated in next commit :)